### PR TITLE
OSDOCS-4669:adds zstream update

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-4669](https://issues.redhat.com//browse/OSDOCS-4669): adds zstream update for 4.11.20

Version(s):
4.11

Issue:
[Jira](https://issues.redhat.com/browse/OSDOCS-4669)

Link to docs preview:
[Preview](https://53794--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-20)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -3052,4 +3052,32 @@ $ oc adm release info 4.11.18 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.11.20 --pullspecs
+----
+
+[id="ocp-4-11-20-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Google Cloud Platform used a hard-coded list to display regions in the installer survey, which resulted in several supported regions being excluded. With this update, all supported regions are included in the installer survey. (link:https://issues.redhat.com/browse/OCPBUGS-3023[*OCPBUGS-3023*])
```
Comment (jboxman-rh) [inline release_notes/ocp-4-11-release-notes.adoc:3074]: I think technically it isn't _GCP_ that had this issue, but OpenShift, specifically the installer. The installer was missing GCP regions. So this sounds like an issue in Google's GCP, but it was us.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -3052,4 +3052,32 @@ $ oc adm release info 4.11.18 --pullspecs
...
+
+[id="ocp-4-11-20-known-issues"]
+==== Known issues
+* Switching the `spec.endpointPublishingStrategy.loadBalancer.scope` of the default `ingresscontroller` results in degraded Ingress Operator. Consequently, routes that use that endpoint such as the console URL become inaccessible. As a workaround, restarting one of the router pods brings back a couple of instances under the `loadbalancer` back to `inService`.(link:https://issues.redhat.com/browse/OCPBUGS-2554[*OCPBUGS-2554*])
```
Comment (jboxman-rh) [inline release_notes/ocp-4-11-release-notes.adoc:3078]:
```suggestion
* Switching the router scope of the default ingress controller by setting the `spec.endpointPublishingStrategy.loadBalancer.scope` field results in a degraded Ingress Operator. Consequently, routes that use that endpoint such as the web console URL become inaccessible. As a workaround, restarting one of the router pods brings back several instances under the `loadbalancer` back to the `inService` status.(link:https://issues.redhat.com/browse/OCPBUGS-2554[*OCPBUGS-2554*])
```

**Review by jboxman-rh — COMMENTED** [review]
@JoeAldinger, I left a few comments. We can improve the specificity a bit for these.

Thanks!

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-11-release-notes.adoc:3074]: Yeah, I re-read the bug and realized some things were a bit off. Thanks, @jboxman-rh

**Pull request closed without merging**